src/main/java/org/rev317/min/api/methods/Equipment.java

```diff
@@ -0,0 +1,139 @@
+package org.rev317.min.api.methods;
```

Comment:
Why do we add this class?

Reply:
For the new `Menu.sendAction` because the client uses the following desc `I,I,I,I,J,I`